tests/test_extension.py

```diff
@@ -14,7 +15,34 @@ def app():
     obs.init_app(app)
 
     @app.route("/login", methods=["GET"])
-    def login():
+    def loginHandler():
+        if request.form.get("username") == "bad":
+            abort(403)
+        return make_response("", 200)
+
+    @app.route("/error", methods=["GET"])
```

Comment:
I appreciate the need to generate and endpoint that triggers errors, but I don't see where you're calling it in your tests. Am I missing something?

Reply:
I'm calling it in test_metrics_errors, but only for one error code 501. I originally thought to use it also for something from 1xx range and 3xx range, but those are not really errors and cannot be used with abort.

Reply:
sorry, my bad, I didn't read the test properly 😄